src/consensus/softforks.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2009-2015 Eric Lombrozo, The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "chain.h"
+#include "consensus/params.h"
+#include "softforks.h"
+
+using namespace Consensus::SoftForks;
```

Comment:
The additional namespace Softforks seems unnecessarily verbose, just Consensus seems enough.